pkg/api/object_example_test.go

```diff
@@ -42,13 +42,26 @@ func (o *ExampleObject) EndpointURL(ctx context.Context) (*url.URL, error) {
 	return url.Parse("/example/v1")
 }
 
+// GetIdentifier returns the identifier of an object
+// The Makefiles `generate` target creates this method for objects that have a anxcloud:"identifier" tagged field
+// Objects with multiple identifying fields can have this method overwritten to return an operation dependent identifier
+// The provided context might not always have the operation set. In this case the method must return the primary identifier
+func (o *ExampleObject) GetIdentifier(ctx context.Context) (string, error) {
```

Comment:
Maybe we should extend this example for showing what other identifier might be possible, how to retrieve the operation and what we mean with "primary identifier"?

Reply:
Also, these sentences all lack a ending `.`, too.